### PR TITLE
Adjust constants to default to mainnet

### PR DESCRIPTION
**Pull request by alexkroeger**
Adjusted relevant constants to point to mainnet.

Testing:
- [x] Tested Locally

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-export const DEFAULT_MAX_BLOCKS_TO_PULL = 500;
-export const DEFAULT_MAX_BLOCKS_TO_SEARCH = 500;
-export const DEFAULT_CHAIN_ID = 42;
+export const DEFAULT_MAX_BLOCKS_TO_PULL = 100;
+export const DEFAULT_MAX_BLOCKS_TO_SEARCH = 100;
+export const DEFAULT_CHAIN_ID = 1;
 export const DEFAULT_BLOCK_FINALITY_THRESHOLD = 0;
 export const DEFAULT_SECONDS_BETWEEN_RUNS = 15;
-export const DEFAULT_STAKING_PROXY_DEPLOYMENT_TRANSACTION= '0x683501fe77223124cb5d284155825dd0df29edbb70cd9f7315580fade2f8d269';
+export const DEFAULT_STAKING_PROXY_DEPLOYMENT_TRANSACTION= '0x4680e9d59bae9bbde1b0bae0fa5157ceea64ea923f2be434e5da6f5df2bdb907';
```
Comment (fragosti) [inline src/constants.ts]: Should we have the default for this be determined by the CHAIN_ID? So here we wouldn't have "default" but "mainnet" and "kovan" and then use those values in `config.ts` if an ENV value isn't passed.

File: src/constants.ts [hunk 2]
```diff
@@ -1,12 +1,12 @@
 export const DEFAULT_LOCAL_POSTGRES_URI = 'postgresql://user:password@localhost/events';
-export const DEFAULT_FIRST_SEARCH_BLOCK = 14425602;
+export const DEFAULT_FIRST_SEARCH_BLOCK = 8952139;
```
Comment (fragosti) [inline src/constants.ts]: Maybe for this too... since these are hard values to figure out.

File: src/constants.ts (hunk 1 above)
Comment (alexkroeger) [reply src/constants.ts]: good point, I'd prefer that

File: src/constants.ts (hunk 2 above)
Comment (alexkroeger) [reply src/constants.ts]: good point as well

**Pull request merged**